chunjun-connectors/chunjun-connector-kafka/src/main/java/com/dtstack/chunjun/connector/kafka/converter/KafkaColumnConverter.java

```diff
@@ -79,18 +81,22 @@ public KafkaColumnConverter(KafkaConf kafkaConf, List<String> keyTypeList) {
         this.kafkaConf = kafkaConf;
         this.outList = keyTypeList;
         if (DEFAULT_CODEC.defaultValue().equals(kafkaConf.getCodec())) {
-            this.decode = new JsonDecoder();
+            this.decoder = new JsonDecoder();
+        } else if (VALUE_CODEC.defaultValue().equals(kafkaConf.getCodec())) {
+            this.decoder = new ValueDecoder();
         } else {
-            this.decode = new TextDecoder();
+            this.decoder = new TextDecoder();
         }
```

Comment:
Is using the 'switch-case' struct better?

Reply:
agree . I will make a change

Reply:
'flink.configuration.ConfigOption.defaultValue' is not a constant expression. so It can't be written after case.like this
```
case DEFAULT_CODEC.defaultValue:
    this.decoder = decoder
```

So I change DEFAULT_CODEC type from flink.configuration.ConfigOption to String